canonical-web-and-design/ubuntu.com
# Add gsi webinar takeover and engage page

## Done

- Add gsi webinar takeover and engage page
- Fix up /engage to not require a group to add the class (it was always a bit odd as it used groups that were unrelated to the engage pages, just borrowing blog classes)
- Shortened a few engage descriptions
- Fixed a few that were the wrong type

## QA

- Check out this feature branch
- Run the site using the command `./run serve` or `dotrun`
- View the site locally in your web browser at: http://0.0.0.0:8001/, http://0.0.0.0:8001/takeovers, http://0.0.0.0:8001/engage/gsi-webinar-series and http://0.0.0.0:8001/engage
- Run through the following [QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html)
- Compare to the [brief](https://docs.google.com/spreadsheets/d/11lbp6jOe_HCdl2vNkqk0_lsV8bWHOjBp6S84xaZVcRg/edit#gid=1595677042) and [copy doc](https://docs.google.com/document/d/1ZweTXyHO9kIU9aE8pw2_Ni_NoBShM_KseY5H2XtGueg/edit)

## Issue / Card

Fixes #7967

## Screenshots

![image](https://user-images.githubusercontent.com/441217/88371211-80023d00-cd8b-11ea-9241-f66a6f69802d.png)
